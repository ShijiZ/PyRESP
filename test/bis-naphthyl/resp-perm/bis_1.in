test for bis-naphthyl
 &cntrl 
 nmol = 2,
 iqopt = 1,
 ihfree = 1,
 qwt = 0.0005,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 1,
 pwt = 0.0005,
 virtual = 0  
 &end 
    1.0
naphthyl
   -1   29  60
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    6    0  0  0  0  0
    6    0  0  0  0  0
    6    0  0  0  0  0
    6    0  0  0  51
    8    0  0
    8   22  53
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0

    1.0
naphthyl
   -1   29  60
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    6    0  0  0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    6    0  0  0  0  0
    6    0  0  0  0  0
    6    0  0  0  0  0
    6    0  0  0  51
    8    0  0
    8   22  53
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0
    1    0  0

   13  -1.000
    1   19    1   20    1   21    1   22    1   23    1   24    1   25    1   26
    1   27    2   20    2   25    2   26    2   27

    2
    1    1    2    1
    2
    1    2    2    2
    2
    1    3    2    3
    2
    1    4    2    4
    2
    1    5    2    5
    2
    1    6    2    6
    2
    1    7    2    7
    2
    1    8    2    8
    2
    1    9    2    9
    2
    1   10    2   10
    2
    1   11    2   11
    2
    1   12    2   12
    2
    1   13    2   13
    2
    1   14    2   14
    2
    1   15    2   15
    2
    1   16    2   16
    2
    1   17    2   17
    2
    1   18    2   18
    2
    1   19    2   19
    2
    1   20    2   20
    2
    1   21    2   21
    2
    1   22    2   22
    2
    1   23    2   23
    2
    1   24    2   24
    2
    1   25    2   25
    2
    1   26    2   26
    2
    1   27    2   27
    2
    1   28    2   28
    2
    1   29    2   29

    2
    1    1    2    1
    2
    1    2    2    2
    2
    1    3    2    3
    2
    1    4    2    4
    2
    1    5    2    5
    2
    1    6    2    6
    2
    1    7    2    7
    2
    1    8    2    8
    2
    1    9    2    9
    2
    1   10    2   10
    2
    1   11    2   11
    2
    1   12    2   12
    2
    1   13    2   13
    2
    1   14    2   14
    2
    1   15    2   15
    2
    1   16    2   16
    2
    1   17    2   17
    2
    1   18    2   18
    2
    1   19    2   19
    2
    1   20    2   20
    2
    1   21    2   21
    2
    1   22    2   22
    2
    1   23    2   23
    2
    1   24    2   24
    2
    1   25    2   25
    2
    1   26    2   26
    2
    1   27    2   27
    2
    1   28    2   28
    2
    1   29    2   29
    2
    1   30    2   30
    2
    1   31    2   31
    2
    1   32    2   32
    2
    1   33    2   33
    2
    1   34    2   34
    2
    1   35    2   35
    2
    1   36    2   36
    2
    1   37    2   37
    2
    1   38    2   38
    2
    1   39    2   39
    2
    1   40    2   40
    2
    1   41    2   41
    2
    1   42    2   42
    2
    1   43    2   43
    2
    1   44    2   44
    2
    1   45    2   45
    2
    1   46    2   46
    2
    1   47    2   47
    2
    1   48    2   48
    2
    1   49    2   49
    2
    1   50    2   50
    2
    1   51    2   51
    2
    1   52    2   52
    2
    1   53    2   53
    2
    1   54    2   54
    2
    1   55    2   55
    2
    1   56    2   56
    2
    1   57    2   57
    2
    1   58    2   58
    2
    1   59    2   59
    2
    1   60    2   60
