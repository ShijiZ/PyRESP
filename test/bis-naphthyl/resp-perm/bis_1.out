
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for bis-naphthyl
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 2   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0
 ipermdip    = 1   pwt         = 0.00050000
 virtual     = 0

 --------------------
 Molecule Information
 --------------------
 Multiple-molecule run of 2 molecules

 Molecule 1 weight:     1.000
 Molecule 1 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
 Number of permanent dipoles: 60
    1    6    0    0    0    0
    2    6    0    0    0    0
    3    6    0    0    0    0
    4    6    0    0    0    0
    5    6    0    0    0    0
    6    6    0    0    0    0
    7    6    0    0    0    0
    8    6    0    0    0    0
    9    6    0    0    0    0
   10    6    0    0    0    0
   11    1    0    0
   12    1    0    0
   13    1    0    0
   14    1    0    0
   15    1    0    0
   16    1    0    0
   17    1    0    0
   18    6    0    0    0    0    0
   19    6    0    0    0    0    0
   20    6    0    0    0    0    0
   21    6    0    0    0   51
   22    8    0    0
   23    8   22   53
   24    1    0    0
   25    1    0    0
   26    1    0    0
   27    1    0    0
   28    1    0    0
   29    1    0    0

 Molecule 2 weight:     1.000
 Molecule 2 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
 Number of permanent dipoles: 60
   30    6    0    0    0    0
   31    6    0    0    0    0
   32    6    0    0    0    0
   33    6    0    0    0    0
   34    6    0    0    0    0
   35    6    0    0    0    0
   36    6    0    0    0    0
   37    6    0    0    0    0
   38    6    0    0    0    0
   39    6    0    0    0    0
   40    1    0    0
   41    1    0    0
   42    1    0    0
   43    1    0    0
   44    1    0    0
   45    1    0    0
   46    1    0    0
   47    6    0    0    0    0    0
   48    6    0    0    0    0    0
   49    6    0    0    0    0    0
   50    6    0    0    0   51
   51    8    0    0
   52    8   22   53
   53    1    0    0
   54    1    0    0
   55    1    0    0
   56    1    0    0
   57    1    0    0
   58    1    0    0
 ---------------------------------------------------------
 Inter-Molecular Charge Constraint(s) Info
 ---------------------------------------------------------
    1   19    1   20    1   21    1   22    1   23    1   24    1   25    1   26
    1   27    2   20    2   25    2   26    2   27

 -------------------------------
 Charges Equivalence Information 
 -------------------------------
    1    1    2    1
    1    2    2    2
    1    3    2    3
    1    4    2    4
    1    5    2    5
    1    6    2    6
    1    7    2    7
    1    8    2    8
    1    9    2    9
    1   10    2   10
    1   11    2   11
    1   12    2   12
    1   13    2   13
    1   14    2   14
    1   15    2   15
    1   16    2   16
    1   17    2   17
    1   18    2   18
    1   19    2   19
    1   20    2   20
    1   21    2   21
    1   22    2   22
    1   23    2   23
    1   24    2   24
    1   25    2   25
    1   26    2   26
    1   27    2   27
    1   28    2   28
    1   29    2   29

 -----------------------------------------
 Permanent Dipoles Equivalence Information
 -----------------------------------------
    1    1    2    1
    1    2    2    2
    1    3    2    3
    1    4    2    4
    1    5    2    5
    1    6    2    6
    1    7    2    7
    1    8    2    8
    1    9    2    9
    1   10    2   10
    1   11    2   11
    1   12    2   12
    1   13    2   13
    1   14    2   14
    1   15    2   15
    1   16    2   16
    1   17    2   17
    1   18    2   18
    1   19    2   19
    1   20    2   20
    1   21    2   21
    1   22    2   22
    1   23    2   23
    1   24    2   24
    1   25    2   25
    1   26    2   26
    1   27    2   27
    1   28    2   28
    1   29    2   29
    1   30    2   30
    1   31    2   31
    1   32    2   32
    1   33    2   33
    1   34    2   34
    1   35    2   35
    1   36    2   36
    1   37    2   37
    1   38    2   38
    1   39    2   39
    1   40    2   40
    1   41    2   41
    1   42    2   42
    1   43    2   43
    1   44    2   44
    1   45    2   45
    1   46    2   46
    1   47    2   47
    1   48    2   48
    1   49    2   49
    1   50    2   50
    1   51    2   51
    1   52    2   52
    1   53    2   53
    1   54    2   54
    1   55    2   55
    1   56    2   56
    1   57    2   57
    1   58    2   58
    1   59    2   59
    1   60    2   60

 -----------------------------
 Atom/Dipole Ivary Information
 -----------------------------
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    6    0    0    0    0
    1    0    0
    1    0    0
    1    0    0
    1    0    0
    1    0    0
    1    0    0
    1    0    0
    6    0    0    0    0    0
    6    0    0    0    0    0
    6    0    0    0    0    0
    6    0    0    0   51
    8    0    0
    8   22   53
    1    0    0
    1    0    0
    1    0    0
    1    0    0
    1    0    0
    1    0    0

    6    1    1    2    3
    6    2    4    5    6
    6    3    7    8    9
    6    4   10   11   12
    6    5   13   14   15
    6    6   16   17   18
    6    7   19   20   21
    6    8   22   23   24
    6    9   25   26   27
    6   10   28   29   30
    1   11   31
    1   12   32
    1   13   33
    1   14   34
    1   15   35
    1   16   36
    1   17   37
    6   18   38   39   40   41
    6   19   42   43   44   45
    6   20   46   47   48   49
    6   21   50   51   51
    8   22   53
    8   22   53
    1   24   55
    1   25   56
    1   26   57
    1   27   58
    1   28   59
    1   29   60

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   58
 Total number of permanent dipoles      =  120
 Weight factor on initial charge restraints=  0.000500
 Weight factor on initial dipole restraints=  0.000500
 There are  3 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 Reading esp's for molecule   2
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 and p0 values will be set to 0

 Number of unique UNfrozen charge centers =    28
 Number of unique UNfrozen permanent dipoles =    58

 Non-linear optimization requested.
  qchnge =    3.5710349780E-02
  qchnge =    1.0980569986E-02
  qchnge =    3.3873047437E-03
  qchnge =    1.4533883331E-03
  qchnge =    7.6061326295E-04
  qchnge =    4.3876135611E-04
  qchnge =    2.6162414050E-04
  qchnge =    1.5763096119E-04
  qchnge =    9.5294160538E-05
  qchnge =    5.7676230068E-05
  qchnge =    3.4924838826E-05
  qchnge =    2.1153754381E-05
  qchnge =    1.2815240688E-05
  qchnge =    7.7649643879E-06
  qchnge =    4.7056105274E-06
  qchnge =    2.8519848087E-06
  qchnge =    1.7287192982E-06
  qchnge =    1.0479483993E-06
  qchnge =    6.3531128281E-07

 Convergence in   18 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: naphthyl
 molecule 2: naphthyl

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6        0.000000     -0.029084      0       0.004801
    2         6        0.000000     -0.002048      0       0.004999
    3         6        0.000000     -0.053028      0       0.004417
    4         6        0.000000     -0.118641      0       0.003222
    5         6        0.000000     -0.133366      0       0.003000
    6         6        0.000000     -0.067664      0       0.004141
    7         6        0.000000      0.001516      0       0.004999
    8         6        0.000000     -0.045949      0       0.004543
    9         6        0.000000     -0.063998      0       0.004211
   10         6        0.000000     -0.030421      0       0.004784
   11         1        0.000000      0.118413      0       0.000000
   12         1        0.000000      0.128482      0       0.000000
   13         1        0.000000      0.059191      0       0.000000
   14         1        0.000000      0.060420      0       0.000000
   15         1        0.000000      0.104603      0       0.000000
   16         1        0.000000      0.066120      0       0.000000
   17         1        0.000000      0.009812      0       0.000000
   18         6        0.000000     -0.011758      0       0.004966
   19         6        0.000000      0.010286      0       0.004974
   20         6        0.000000     -0.016195      0       0.004936
   21         6        0.000000      0.145558      0       0.002831
   22         8        0.000000     -0.666554      0       0.000742
   23         8        0.000000     -0.666554     22       0.000742
   24         1        0.000000      0.049401      0       0.000000
   25         1        0.000000     -0.016027      0       0.000000
   26         1        0.000000      0.016093      0       0.000000
   27         1        0.000000      0.080061      0       0.000000
   28         1        0.000000     -0.000686      0       0.000000
   29         1        0.000000      0.072015      0       0.000000

   30         6        0.000000     -0.029084      1       0.004801
   31         6        0.000000     -0.002048      2       0.004999
   32         6        0.000000     -0.053028      3       0.004417
   33         6        0.000000     -0.118641      4       0.003222
   34         6        0.000000     -0.133366      5       0.003000
   35         6        0.000000     -0.067664      6       0.004141
   36         6        0.000000      0.001516      7       0.004999
   37         6        0.000000     -0.045949      8       0.004543
   38         6        0.000000     -0.063998      9       0.004211
   39         6        0.000000     -0.030421     10       0.004784
   40         1        0.000000      0.118413     11       0.000000
   41         1        0.000000      0.128482     12       0.000000
   42         1        0.000000      0.059191     13       0.000000
   43         1        0.000000      0.060420     14       0.000000
   44         1        0.000000      0.104603     15       0.000000
   45         1        0.000000      0.066120     16       0.000000
   46         1        0.000000      0.009812     17       0.000000
   47         6        0.000000     -0.011758     18       0.004966
   48         6        0.000000      0.010286     19       0.004974
   49         6        0.000000     -0.016195     20       0.004936
   50         6        0.000000      0.145558     21       0.002831
   51         8        0.000000     -0.666554     22       0.000742
   52         8        0.000000     -0.666554     22       0.000742
   53         1        0.000000      0.049401     24       0.000000
   54         1        0.000000     -0.016027     25       0.000000
   55         1        0.000000      0.016093     26       0.000000
   56         1        0.000000      0.080061     27       0.000000
   57         1        0.000000     -0.000686     28       0.000000
   58         1        0.000000      0.072015     29       0.000000

 Sum over the calculated charges:     -2.000

       Permanent Dipoles Before & After Optimization
 dip.no   element.no   p(init)        p(opt)     ivary    d(rstr)/dp
    1         6        0.000000      0.039307      0       0.004653
    2         6        0.000000     -0.013051      0       0.004958
    3         6        0.000000     -0.023297      0       0.004870
    4         6        0.000000     -0.036975      0       0.004690
    5         6        0.000000     -0.006247      0       0.004990
    6         6        0.000000      0.046252      0       0.004538
    7         6        0.000000      0.012321      0       0.004962
    8         6        0.000000     -0.011353      0       0.004968
    9         6        0.000000     -0.001485      0       0.004999
   10         6        0.000000     -0.025163      0       0.004849
   11         6        0.000000     -0.028757      0       0.004805
   12         6        0.000000      0.060285      0       0.004282
   13         6        0.000000     -0.029875      0       0.004791
   14         6        0.000000     -0.029914      0       0.004790
   15         6        0.000000      0.069109      0       0.004113
   16         6        0.000000     -0.027215      0       0.004825
   17         6        0.000000      0.012463      0       0.004962
   18         6        0.000000      0.014232      0       0.004950
   19         6        0.000000      0.027800      0       0.004817
   20         6        0.000000     -0.000049      0       0.005000
   21         6        0.000000     -0.026744      0       0.004830
   22         6        0.000000      0.008596      0       0.004982
   23         6        0.000000     -0.030995      0       0.004776
   24         6        0.000000      0.021929      0       0.004884
   25         6        0.000000     -0.032052      0       0.004761
   26         6        0.000000     -0.010040      0       0.004975
   27         6        0.000000      0.045995      0       0.004543
   28         6        0.000000     -0.070048      0       0.004095
   29         6        0.000000     -0.001501      0       0.004999
   30         6        0.000000      0.073953      0       0.004020
   31         1        0.000000     -0.016643      0       0.000000
   32         1        0.000000     -0.091857      0       0.000000
   33         1        0.000000     -0.137475      0       0.000000
   34         1        0.000000     -0.141183      0       0.000000
   35         1        0.000000     -0.086064      0       0.000000
   36         1        0.000000     -0.136647      0       0.000000
   37         1        0.000000     -0.180286      0       0.000000
   38         6        0.000000      0.037564      0       0.004681
   39         6        0.000000      0.011906      0       0.004965
   40         6        0.000000     -0.053748      0       0.004404
   41         6        0.000000     -0.006297      0       0.004990
   42         6        0.000000      0.004784      0       0.004994
   43         6        0.000000      0.048397      0       0.004501
   44         6        0.000000     -0.062939      0       0.004232
   45         6        0.000000      0.006104      0       0.004991
   46         6        0.000000      0.026650      0       0.004831
   47         6        0.000000     -0.008856      0       0.004981
   48         6        0.000000     -0.006606      0       0.004989
   49         6        0.000000     -0.012085      0       0.004964
   50         6        0.000000      0.153769      0       0.002726
   51         6        0.000000     -0.038328      0       0.004669
   52         6        0.000000     -0.038328     51       0.004669
   53         8        0.000000      0.111278      0       0.003342
   54         8        0.000000      0.111278     53       0.003342
   55         1        0.000000      0.030482      0       0.000000
   56         1        0.000000     -0.072135      0       0.000000
   57         1        0.000000     -0.001827      0       0.000000
   58         1        0.000000      0.009147      0       0.000000
   59         1        0.000000     -0.076412      0       0.000000
   60         1        0.000000      0.021399      0       0.000000

   61         6        0.000000      0.039307      1       0.004653
   62         6        0.000000     -0.013051      2       0.004958
   63         6        0.000000     -0.023297      3       0.004870
   64         6        0.000000     -0.036975      4       0.004690
   65         6        0.000000     -0.006247      5       0.004990
   66         6        0.000000      0.046252      6       0.004538
   67         6        0.000000      0.012321      7       0.004962
   68         6        0.000000     -0.011353      8       0.004968
   69         6        0.000000     -0.001485      9       0.004999
   70         6        0.000000     -0.025163     10       0.004849
   71         6        0.000000     -0.028757     11       0.004805
   72         6        0.000000      0.060285     12       0.004282
   73         6        0.000000     -0.029875     13       0.004791
   74         6        0.000000     -0.029914     14       0.004790
   75         6        0.000000      0.069109     15       0.004113
   76         6        0.000000     -0.027215     16       0.004825
   77         6        0.000000      0.012463     17       0.004962
   78         6        0.000000      0.014232     18       0.004950
   79         6        0.000000      0.027800     19       0.004817
   80         6        0.000000     -0.000049     20       0.005000
   81         6        0.000000     -0.026744     21       0.004830
   82         6        0.000000      0.008596     22       0.004982
   83         6        0.000000     -0.030995     23       0.004776
   84         6        0.000000      0.021929     24       0.004884
   85         6        0.000000     -0.032052     25       0.004761
   86         6        0.000000     -0.010040     26       0.004975
   87         6        0.000000      0.045995     27       0.004543
   88         6        0.000000     -0.070048     28       0.004095
   89         6        0.000000     -0.001501     29       0.004999
   90         6        0.000000      0.073953     30       0.004020
   91         1        0.000000     -0.016643     31       0.000000
   92         1        0.000000     -0.091857     32       0.000000
   93         1        0.000000     -0.137475     33       0.000000
   94         1        0.000000     -0.141183     34       0.000000
   95         1        0.000000     -0.086064     35       0.000000
   96         1        0.000000     -0.136647     36       0.000000
   97         1        0.000000     -0.180286     37       0.000000
   98         6        0.000000      0.037564     38       0.004681
   99         6        0.000000      0.011906     39       0.004965
  100         6        0.000000     -0.053748     40       0.004404
  101         6        0.000000     -0.006297     41       0.004990
  102         6        0.000000      0.004784     42       0.004994
  103         6        0.000000      0.048397     43       0.004501
  104         6        0.000000     -0.062939     44       0.004232
  105         6        0.000000      0.006104     45       0.004991
  106         6        0.000000      0.026650     46       0.004831
  107         6        0.000000     -0.008856     47       0.004981
  108         6        0.000000     -0.006606     48       0.004989
  109         6        0.000000     -0.012085     49       0.004964
  110         6        0.000000      0.153769     50       0.002726
  111         6        0.000000     -0.038328     51       0.004669
  112         6        0.000000     -0.038328     51       0.004669
  113         8        0.000000      0.111278     53       0.003342
  114         8        0.000000      0.111278     53       0.003342
  115         1        0.000000      0.030482     55       0.000000
  116         1        0.000000     -0.072135     56       0.000000
  117         1        0.000000     -0.001827     57       0.000000
  118         1        0.000000      0.009147     58       0.000000
  119         1        0.000000     -0.076412     59       0.000000
  120         1        0.000000      0.021399     60       0.000000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          3.2877896E+01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   7.9098013E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.7787413E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.5510678E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1       12.94296  -10.96492    6.21250   -2.94881
 #MOL         D          Dx         Dy         Dz
   2       12.94296  -10.96492    6.21250   -2.94881

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X  -20.97918
        Y   17.68469   11.11231
        Z  -11.58269    4.50311    9.86686
 #MOL          X          Y          Z
   2    X  -20.97918
        Y   17.68469   11.11231
        Z  -11.58269    4.50311    9.86686

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   19.00140
        Y    0.00000   13.45311
        Z    0.00000    0.00000  -32.45451
 #MOL          X          Y          Z
   2    X   19.00140
        Y    0.00000   13.45311
        Z    0.00000    0.00000  -32.45451